nbflow/example/SConstruct

```diff
@@ -2,4 +2,8 @@ import os
 from nbflow.scons import setup
 
 env = Environment(ENV=os.environ)
-setup(env, ["analyses"])
+timeout = ARGUMENTS.get('timeout', 0)
```

Comment:
Ideally this is something that would be handled by the nbflow code, rather than something users have to put in the SConstruct file. Maybe pass the `ARGUMENTS` variable to the `setup` function directly and let `setup` parse the timeout?

Reply:
The next commit fixes this parsing the `ARGUMENTS` inside `setup`